xml/System/Convert.xml

```diff
@@ -1208,7 +1208,7 @@
         <param name="inArray">An array of 8-bit unsigned integers.</param>
         <param name="options">
           <see cref="F:System.Base64FormattingOptions.InsertLineBreaks" /> to insert a line break every 76 characters, or <see cref="F:System.Base64FormattingOptions.None" /> to not insert line breaks.</param>
-        <summary>Converts an array of 8-bit unsigned integers to its equivalent string representation that is encoded with base-64 digits. A parameter specifies whether to insert line breaks in the return value.</summary>
+        <summary>Converts an array of 8-bit unsigned integers to its equivalent string representation that is encoded with base-64 digits, allowing to specify whether to insert line breaks in the return value.</summary>
```

Comment:
Unlike the other newer APIs, this one does not have a default value. The APIs below say "allowing to optionally specify" for this reason.
I also thought that it sounded strange to say "a parameter specifies". Normally we don't mention parameters that way.

Reply:
Would something like this read better?

```suggestion
        <summary>Converts an array of 8-bit unsigned integers to its equivalent string representation that is encoded with base-64 digits, and allows you to specify whether to insert line breaks in the return value.</summary>
```

Reply:
It sounds kind of strange when the MS Docs page is talking to "you". I don't recall seeing this kind of messaging before. I think I want to stay with the one I wrote if that's ok. Same with the suggestion below.

Reply:
Actually, if the purpose is to break up the sentences because they are too long, it makes more sense. I'll apply them.

Reply:
I think it's less common on the API docs but we've been using fairly extensively on the conceptual docs.

I think the original sentence is not good the way it's written because it seems that the action of converting is what is allowing you to specify whether to insert line breaks or not. You could also just remove this part since it's captured in the parameter sections and you don't have extra overloads.